support-frontend/assets/components/blockLabel/blockLabel.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+
+import React, { type Node } from 'react';
+import { css } from '@emotion/core';
+import { space } from '@guardian/src-foundations';
+import { headline } from '@guardian/src-foundations/typography';
+import { neutral, text } from '@guardian/src-foundations/palette';
+import { from } from '@guardian/src-foundations/mq';
+
+const blockLabel = css`
+  display: inline-block;
+  padding: ${space[1]}px ${space[2]}px;
+  ${headline.xxxsmall({ fontWeight: 'bold' })};
+  background-color: ${neutral[0]};
+  color: ${text.ctaPrimary};
+  margin-left: -1px;
+
+  ${from.tablet} {
+    ${headline.xxsmall({ fontWeight: 'bold' })};
+  }
+
+  ${from.desktop} {
+    ${headline.xsmall({ fontWeight: 'bold' })};
+  }
+`;
+
+type PropTypes = {|
+  children: Node;
+  tag?: string;
+  cssOverrides?: string | string[];
+|};
+
+function BlockLabel({ children, tag = 'div', cssOverrides }: PropTypes) {
```

Comment:
A question regarding `defaultProps` and the default values of the params. The default prop/param for `tag` both match, so is there a reason why we'd not set the default parameter value for `cssOverrides` to `''` when we state the default prop value is `''`?

Reply:
It seems to be a Flow/ESLint/something else issue. Setting the `defaultProps` is required for optional props, but if you don't provide a default value when destructuring or write conditional logic around an optional prop (despite it having a non-falsey default value), you get a Flow error, because it can't see the `defaultProps`.

I've had to do this in a few other places in the codebase, I am 99% sure we can get rid of it once we migrate to Typescript.